lib/__test__/rule-list.test.js

```diff
@@ -181,12 +188,14 @@ describe('rules', () => {
 
     suite('common denominators', rules.COMMON_DENOMINATOR, [
         ['2/6 + 1/4', '(2 * 2) / (6 * 2) + (1 * 3) / (4 * 3)'],
+        ['2/(3 + x) + 3/(2 + x)', '(2 * (2 + x)) / ((3 + x) * (2 + x)) + (3 * (3 + x)) / ((3 + x) * (2 + x))'],
```

Comment:
There are a couple of options:
- don't include non-fractions when matching
- include non-fractions, but convert them to fractions before computing the common denominator.

You should ask Gen for their input on this.

What happens if some of the fractions have the same denominator while others don't?